lib/puppet/util/execution.rb

```diff
@@ -98,11 +98,11 @@ def self.exitstatus
   # @raise [exception] under same conditions as {execute}, but raises the given `exception` with the output as argument
   # @return (see execute)
   # @api public
+  # @deprecated
```

Comment:
@Iristyle I deprecated the provider wrappers and modified the rpm provider to not call the deprecated method.